src/pages/Welcome/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from 'react-router-dom';
+import {
+  Button, Logo, Subtitle, Title,
+} from '../../components/common';
```

Comment:
Not asking for this change for this PR but it would be a very good idea to implement absolute imports in this project. The sooner the better so the transition is easier. We usually use absolute imports in all of our projects because it's much easier to know from where exactly something is being imported, and if you move a file you won't have to update all of its imports, just the places where it is being imported.